.gitignore

```diff
@@ -18,13 +18,18 @@ project/boot/
 project/plugins/project/
 .bsp
 .metals
+metals.sbt
```

Comment:
Just curious what these are for?

Reply:
https://scalameta.org/metals/blog/2020/08/19/lithium#sbt-script-support

It is metals-related internal thing.